libqtile/layout/zoomy.py

```diff
@@ -138,11 +138,14 @@ def info(self):
         return d
 
     def focus(self, win):
-        if self.focused and self.property_name:
+        if self.focused and self.property_name and self.focused.window.get_property(
```

Comment:
Ok. Tomorrow in the morning I'll fix this
2015 Vas 12 19:39 "Tycho Andersen" <notifications@github.com> rašė:

> In libqtile/layout/zoomy.py
> <https://github.com/qtile/qtile/pull/607#discussion_r24600443>:
>
> > @@ -138,11 +138,14 @@ def info(self):
> >          return d
> >
> >      def focus(self, win):
> > -        if self.focused and self.property_name:
> > +        if self.focused and self.property_name and self.focused.window.get_property(
>
> Can we make this is None, that way people can set it to "" if they want?
> Other than that, this patch looks good to me.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/qtile/qtile/pull/607/files#r24600443>.
>